src/platform/qt/utils.cpp

```diff
@@ -29,18 +33,25 @@ QString niceSizeFormat(size_t filesize) {
 	return unit.arg(size, 0, 'f', int(size * 10) % 10 ? 1 : 0);
 }
 
-QString nicePlatformFormat(mPlatform platform) {
+QString nicePlatformFormat(mPlatform platform, int validModels) {
 	switch (platform) {
 #ifdef M_CORE_GBA
 	case mPLATFORM_GBA:
-		return QObject::tr("GBA");
```

Comment:
Why get rid of the translation?

Reply:
Because these are used directly as hash keys, so translating them would break it.